src/System.Security.Principal.Windows/src/System/Security/Principal/NTAccount.cs

```diff
@@ -142,19 +142,7 @@ public override IdentityReference Translate(Type targetType)
 
         public override bool Equals(object o)
         {
-            if (o == null)
-            {
-                return false;
-            }
-
-            NTAccount nta = o as NTAccount;
-
-            if (nta == null)
-            {
-                return false;
-            }
-
-            return (this == nta); // invokes operator==
+            return (this == o as NTAccount); // invokes operator==
```

Comment:
My brain-parser has the opposite problem, and gets tripped wondering if extra parentheses means something special is happening with precedence.